packages/foundation-models/README.md

```diff
@@ -154,6 +141,28 @@ const embedding = response.getEmbedding();
 
 Define optional request configuration when running the client, similar to the chat client.
 
+### Custom Request Configuration
+
+Set custom request configuration in the `requestConfig` parameter when calling the `run` function of a chat or embedding client.
```

Comment:
```suggestion
Set custom request configuration in the `requestConfig` parameter when calling the `run()` method of a chat or embedding client.
```
[styleguide] Use [Code Format for Method Names](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-code-format-for-method-names) and [Use Function vs Method Accordingly](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-function-vs-method-accordingly)